app/Console/Commands/PostQuantity.php

```diff
@@ -74,23 +68,17 @@ public function handle()
                     $missingQuantity = $signup->getQuantity() - $postQuantityTotal;
                     $this->putQuantityOnPosts($posts, $missingQuantity);
 
-                    // Advance the progress bar
-                    $bar->advance();
-
                     // Move on to the next signup
                     continue;
                 }
 
                 // If no posts have a quantity yet
                 $quantity = $signup->getQuantity();
                 $this->putQuantityOnPosts($posts, $quantity);
-
-                // Advance the progress bar
-                $bar->advance();
             }
+            info('rogue:postquantity: Updated posts for signup ' . $signup->id);
```

Comment:
are you going to have a `$signup` at this point? it's not in the foreach loop.

Reply:
@sbsmith86 I wasn't sure if it would still be set there either, but it worked locally

Reply:
hmm is it for sure the 1000th signup or just the first one in that chunk?

Reply:
@sbsmith86 it's the 1000th signup and the last one in the chunk. here is the last bit of example logs from my local:
```
[2018-01-22 16:14:23] local.INFO: rogue:postquantity: Updated posts for signup 27000  {"user_id":null,"client_id":null,"request_id":null}
[2018-01-22 16:14:23] local.INFO: rogue:postquantity: Updated posts for signup 28000  {"user_id":null,"client_id":null,"request_id":null}
[2018-01-22 16:14:23] local.INFO: rogue:postquantity: Updated posts for signup 29000  {"user_id":null,"client_id":null,"request_id":null}
[2018-01-22 16:14:23] local.INFO: rogue:postquantity: Updated posts for signup 29556  {"user_id":null,"client_id":null,"request_id":null}
```
So the last chunk had less than 1000 signups so the signup id is `29556`